recipe/meta.yaml

```diff
@@ -31,7 +31,9 @@ requirements:
     - scipy >=1.4.0
     - spiceypy >=4.0.1
     - pyyaml
-
+  run_contrained: 
+    - gdal 
```

Comment:
will only install if it can do so without conflicts. important to co-install with ISIS.